src/app/core/site-settings/site-settings.service.spec.ts

```diff
@@ -44,36 +42,42 @@ describe("SiteSettingsService", () => {
     expect(service).toBeTruthy();
   });
 
-  it("should only publish changes if property has changed", () => {
+  it("should only publish changes if property has changed", fakeAsync(() => {
```

Comment:
Where does this change come from?

Reply:
I think this was an attempt to fix this unstable test. Has nothing to do with this PR however, will remove.